src/state/slices/tradeQuoteSlice/tradeQuoteSlice.ts

```diff
@@ -22,11 +23,15 @@ export const tradeQuoteSlice = createSlice({
     setSwapperName: (state, action: PayloadAction<SwapperName | undefined>) => {
       state.swapperName = action.payload
     },
-    setQuote: (state, action: PayloadAction<TradeQuote2 | undefined>) => {
-      state.quote = action.payload
-    },
-    setError: (state, action: PayloadAction<SwapErrorRight | undefined>) => {
-      state.error = action.payload
+    setQuotes: (state, action: PayloadAction<{ quotes: ApiQuote[]; queryStartTime: number }>) => {
+      if (
+        !state.lastRequestStartTime ||
+        action.payload.queryStartTime > state.lastRequestStartTime
+      ) {
+        state.quotes = action.payload.quotes
+        state.swapperName = action.payload.quotes[0]?.swapperName
```

Comment:
does this mean every time it polls it will reset your swapper selection?

Reply:
Ugh, it does. We don't have this problem in prod because at some point polling was removed.

Proposing I fix this in a subsequent PR to maintain velocity.

Reply:
im for this, maybe make it the next PR because testing will be tricky with it resetting

Reply:
We can always bump the polling interval to something like 60 seconds in the interim.